crypto/fipsmodule/ec/ec_key.c

```diff
@@ -540,10 +540,10 @@ int EC_KEY_generate_key_fips(EC_KEY *eckey) {
   eckey->priv_key = NULL;
 
 #if defined(AWSLC_FIPS)
```

Comment:
Now that we have a function that can handle fips-mode or not, would prefer to push that stuff to the function.

But I think it would also need an additional input to signal whether it should abort or not when in non-fips mode. Which obviously complicates that a bit...